spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -49,9 +60,29 @@
 
 	private final SubscriberFactory subscriberFactory;
 
-	public PubSubTemplate(PublisherFactory publisherFactory, SubscriberFactory subscriberFactory) {
+	private final SubscriptionAdminSettings pullSettings;
```

Comment:
The type doesn't seem right. Should it be `UnaryCallSettings<PullRequest, PullResponse>`? `SubscriptionAdminSettings` has a lot of other configuration that is not relevant to pull settings.

Reply:
Why do you say so?

`GrpcSusbcriberStub.create()` takes in a `SubscriptionAdminSettings`.